### PR TITLE
fix(e2ee) discards frames that cannot be decrypted

### PR DESCRIPTION
**Pull request by tmoldovan8x8**

### CODE REVIEW

File: modules/e2ee/Context.js [hunk 1]
```diff
@@ -253,11 +249,9 @@ export class Context {
             newUint8.set(new Uint8Array(plainText), frameHeader.byteLength);
 
             encodedFrame.data = newData;
-        } catch (error) {
-            if (this._sharedKey) {
```
Comment (saghul) [inline modules/e2ee/Context.js:257]: Why was this check for shared key performed here?

**Review by saghul — DISMISSED** [review]
LGTM with a question.

File: modules/e2ee/Context.js (hunk 1 above)
Comment (tmoldovan8x8) [reply modules/e2ee/Context.js:257]: I think cause the rotation should be handled externally in sharedKey mode.

Comment (saghul) [reply modules/e2ee/Context.js:257]: Then shouldn't we just return in case of a shared key? We don't want to ratchet in that case.

Comment (tmoldovan8x8) [reply modules/e2ee/Context.js:257]: Yeah, true that. We should not return the encoded frame, but still return from the method.

**Review by saghul — APPROVED** [review]

**Pull request merged**